src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2359,8 +2420,30 @@ public void HandleIDebugModuleLoadEvent2(IDebugEngine2 pEngine, IDebugProcess2 p
             string moduleLoadMessage = null;
             int isLoad = 0;
             ((IDebugModuleLoadEvent2)pEvent).GetModule(out module, ref moduleLoadMessage, ref isLoad);
-
+            
             m_logger.WriteLine(LoggingCategory.Module, moduleLoadMessage);
+
+            var debugModuleInfos = new MODULE_INFO[1];
+            if (module.GetInfo(enum_MODULE_INFO_FIELDS.MIF_ALLFIELDS, debugModuleInfos) == HRConstants.S_OK)
```

Comment:
Create a helper method to convert a MODULE_INFO into a ProtocolMessages.Module to remove duplicated code.